tests/test_cachito_util.py

```diff
@@ -63,8 +63,8 @@ def handle_request_sources(http_request):
     response = api.request_sources(CACHITO_REQUEST_REPO, CACHITO_REQUEST_REF, **additional_params)
     assert response['id'] == CACHITO_REQUEST_ID
 
-    response_json = json.dumps(response_data, indent=4)
-    assert 'Cachito response:\n{}'.format(response_json) in caplog.text
+    response_json = 'Cachito response:\n{}'.format(json.dumps(response_data, indent=4))
+    assert "".join(response_json.split()) in "".join(caplog.text.split())
```

Comment:
Could be this written in more readable way?
```
>>> "something with many          whitespaces".replace(" ", "")                                                                                                                                                                            
'somethingwithmanywhitespaces'
```

Please put a comment above, why this change is needed.

Reply:
Do I need to put it above each changing?

Reply:
IMHO Yes

Reply:
IMHO, **definitely**; I didn't even realize what was going on here until I saw the comment :slightly_smiling_face: